src/audio/VoiceRecording.ts

```diff
@@ -204,8 +204,7 @@ export class VoiceRecording extends EventEmitter implements IDestroyable {
         // time needed to encode a sample/frame.
         //
         // Ref for recorderSeconds: https://github.com/chris-rudmin/opus-recorder#instance-fields
-        const recorderSeconds = this.recorder.encodedSamplePosition / 48000;
```

Comment:
some of the comment above this variable needs to move into the getter, as now the `48000` is without context.

Reply:
Comment added to the getter